website/docs/components/code-block/partials/code/how-to-use.md

```diff
@@ -32,6 +32,38 @@ as |CB|>
 </Hds::CodeBlock>
 ```
 
+### Title `tag`
+
+The `tag` value changes the HTML element wrapping the content in the title block. When organizing the content on a webpage, the heading levels should reflect the structure of the page. For example, if a CodeBlock is within a subsection of the page below a heading level 2, the value should be `"h3"`. 
```

Comment:
is it `tag` or `titleTag`?  I'm asking because the matching component api says titleTag.

Reply:
Oops, its `tag`